agentops/client.py

```diff
@@ -212,7 +212,11 @@ def start_session(self, tags: Optional[List[str]] = None, config: Optional[Confi
 
         self._session = Session(uuid4(), tags or self._tags, host_env=get_host_env())
         self._worker = Worker(config or self.config)
-        self._worker.start_session(self._session)
+        start_session_result = self._worker.start_session(self._session)
+        if not start_session_result:
+            self._session = None
+            return logging.warning("AgentOps: Cannot start session")
```

Comment:
We'd be leaving 2 messages when this happens. Do we want to consolidate or do you think that's fine?